### ISSUE TITLE
Reproduction Performance Issue

### ISSUE THREAD

**Issue by BlueCat-de** [hustvl/DiffusionDrive#22]
Hi!
I trained the diffusiondrive agent with diffault cfg from scratch, however, the PDMS is 82 on navtest. The PDMS of the official ckpt is 88, the same as your paper declared. It would helps me a lot if you could share your random seed or any other training techniques, THX!

**Comment by LegendBC** [comment]
Hi @BlueCat-de , thank you for bringing this up, and sorry for the delayed response. The discrepancy you observed is due to migrating misalignment in the training hyperparameters. To address this, I’m currently running experiments using the released code to ensure everything is aligned correctly. I’ll update you within a few days once the issue is resolved.

Thank you for your interest and patience!

**Comment by varunjammula** [comment]
I used a single A-100 with 3e-4 learning rate and got a PDMS score of 87.3

**Comment by BlueCat-de** [comment]
> I used a single A-100 with 3e-4 learning rate with a batch size of 256 and got a PDMS score of 87.3

Wow, I'll try this setting, thanks! By the way, what's the total training time on a single GPU?

**Comment by varunjammula** [comment]
It took approximately 23hrs ish..

**Comment by LegendBC** [comment]
> The PDMS of the official ckpt is 88, the same as your paper declared.

Hi @BlueCat-de , can you help with this issue https://github.com/hustvl/DiffusionDrive/issues/23 raised by @varunjammula ?

**Comment by LegendBC** [comment]
@BlueCat-de @varunjammula I update the training hyper-parameter in the latest code, check it out!

**Comment by tjyuyao** [comment]
@LegendBC Hi, may I ask if you use 8 gpus or a single gpu for training on navtrain ? For a fixed 100 epoch setting, training on 8 gpus means 1/8 of overall update steps compared to training on a single gpu. The transfuser baseline is trained on a single gpu, by the way.
